experimenter/experimenter/experiments/constants.py

```diff
@@ -994,6 +994,38 @@ class FirefoxLabsGroups(models.TextChoices):
 Optional - We believe this outcome will <describe impact> on <core metric>
     """  # noqa
 
+    KPI_AREA = "KPI Metrics"
+    NOTABLE_CHANGES_AREA = "Notable Changes"
+    DEFAULT_METRIC_AREAS = [NOTABLE_CHANGES_AREA, KPI_AREA]
+
+    KPI_METRICS = [
+        {
+            "group": "other_metrics",
+            "friendly_name": "Daily Active Users",
+            "slug": "client_level_daily_active_users_v2",
+            "description": "Daily active users per client.",
+        },
+        {
+            "group": "other_metrics",
+            "friendly_name": "Days of Use",
+            "slug": "days_of_use",
+            "description": "Number of days the browser was used.",
+        },
```

Comment:
`days_of_use` isn't a KPI anymore, but there are older experiments that don't have data for the replacement (`client_level_daily_active_users_v2`). If I remember right, we kept it in the list so we could fall back to it if someone was looking at an old experiment without the `client_level_daily_active_users_v2` data. Are we doing something similar in the new UI?